app/views/favorite/index.html.erb

```diff
@@ -1,8 +1,11 @@
 <h1>Favorited Pets</h1><br><br>
 
- <% favorites.contents.keys.each do |pet| %>
- <%= link_to "#{Pet.find(pet).name}", "/pets/#{pet}" %>
-  <%= image_tag("#{Pet.find(pet).image}", :size => "260x180") %>
+ <% favorites.contents.each do |pet_id, favorite| %>
+
+  <%if favorite >= 1%>
+    <%= link_to "#{Pet.find(pet_id).name}", "/pets/#{pet_id}" %>
```

Comment:
A lot of logic in the view file. Can we push this back to the controller?

Reply:
I dont know of a way to have link_to's sent from the controller to the view, for now I cant think of a way to do this differently